src/main/java/in/ashwanthkumar/gocd/github/provider/github/model/PullRequestStatus.java

```diff
@@ -10,25 +9,18 @@
     private String lastHead;
     // We use this to find changes in a PR
     private String mergeSHA;
-    @Exclude
```

Comment:
After #14 we only store {"pr-number": "latest-SHA"}. Its given as `Map` by git-cmd & stored as is.